src/url_shortener/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
+from .models import Url
 
-# Create your views here.
+def generate_short_url(request):
+    if request.POST:
+        long_url = request.POST.get('url')
+        size =len(Url.objects.all())
+        n = int(size/26) + 1
+        link_ending = ""
+        for i in range(n):
+            num = (size-(i*(size-26))) % 26
+            letter = chr((num-1) + ord('a'))
+            link_ending += letter
+        short_url = "http://pin.y/"+link_ending[::-1]
+        Url.objects.create(url=long_url, short_url=short_url)
+    return render(request, 'url_shortener/seila.html', {})
```

Comment:
Você não criou o template né? Dessa forma, se eu testar aqui, não vou conseguir rodar. Quando for assim, retorna um `HttpResponse(short_url)` por exemplo, que daí eu consigo ver o resultado final :)